pkgs/by-name/pa/paratest/package.nix

```
@@ -0,0 +1,28 @@
+{ php, fetchFromGitHub, lib }:
+
+let
+  phpPackage = php.withExtensions ({ enabled, all }: enabled ++ [ all.pcov ]);
+in php.buildComposerProject (finalAttrs: {
```

I think this is enough, and you can remove the line `php = phpPackages;`.
```suggestion
in (php.withExtensions ({ enabled, all }: enabled ++ [ all.pcov ])).buildComposerProject (finalAttrs: {
```